src/WebJobs.Script.NuGet/WebJobs.Script.nuspec

```diff
@@ -16,12 +16,12 @@
     <dependencies>
       <dependency id="Microsoft.Azure.WebJobs" version="2.0.0-alpha-10331" />
```

Comment:
From myget: 

DEPENDENCIES
Microsoft.Azure.WebJobs (>= 2.0.0-alpha-10331) 

Also latest version is 2.0.0-alpha-10331, unless I am not looking in the right place.